### PR TITLE
Adding ability to return replicaset state as metric

### PR DESCRIPTION
**Pull request by naemono**
Adding ability to return replicaset state as metric
Tests surrounding adding ability to return replicaset state as metric

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose

Tests surrounding adding ability to return replicaset state as metric

#### Known Compatablity Issues

None

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [1.3.0]
 ### Added
+- Support for returning replicaset state metrics (@naemono)
```
Comment (majormoses) [inline CHANGELOG.md]: move this to the unreleased section.

**Comment by majormoses** [comment]
@naemono any chance of fixing up the changelog so this can get merged?

**Comment by majormoses** [comment]
@naemono if you have no intention of fixing this up can you give me access so I can do some fixup for you?

**Comment by majormoses** [comment]
I fixed it up for you.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-mongodb/versions/1.4.0 (sorry forgot to post earlier)

**Comment by naemono** [comment]
Really sorry for missing this guys.  Thanks for finishing it up.

**Pull request merged**